components/org.wso2.carbon.identity.api.server.email.template/org.wso2.carbon.identity.rest.api.server.email.template.v2/src/main/resources/email-template.yml

```diff
@@ -0,0 +1,799 @@
+      responses:
+        '201':
+          description: Item Created
+          headers:
+            self:
+              description: Location of the newly created email template type.
+              schema:
+                type: string
+          content:
+            'application/json':
+              schema:
+                $ref: '#/components/schemas/EmailTemplateType'
+        '400':
+          $ref: '#/components/responses/InvalidInput'
+        '401':
+          $ref: '#/components/responses/Unauthorized'
+        '403':
+          $ref: '#/components/responses/Forbidden'
+        '409':
+          $ref: '#/components/responses/Conflict'
+        '500':
+          $ref: '#/components/responses/ServerError'
+  '/email/template-types/{template-type-id}':
+    get:
+      tags:
+        - Email Template Types
+      summary: Retrieves the email template type corresponding to the template type id.
+      operationId: getEmailTemplateType
+      description: |
+        Retrieves the email template type in the system identified by the template-type-id. <br>
+
+        <b>Permission required:</b> <br>
+        * /permission/admin/manage/identity/emailmgt/view
+        <br><b>Scopes required:</b><br>
+        * internal_email_mgt_view<br>
+      parameters:
+        - $ref: '#/components/parameters/templateTypeIdPathParam'
+      responses:
+        '200':
+          description: Search results matching the given criteria.
+          content:
+            'application/json':
+              schema:
+                $ref: '#/components/schemas/EmailTemplateTypeWithID'
+        '400':
+          $ref: '#/components/responses/InvalidInput'
+        '401':
+          $ref: '#/components/responses/Unauthorized'
+        '403':
+          $ref: '#/components/responses/Forbidden'
+        '404':
+          $ref: '#/components/responses/NotFound'
+        '500':
+          $ref: '#/components/responses/ServerError'
+    delete:
+      tags:
+        - Email Template Types
+      summary: Removes an email template type.
+      operationId: deleteEmailTemplateType
+      description: |
+        Removes an existing email template type with all its email templates
+        from the system. <br>
+
+        <b>Permission required:</b><br>
+        * /permission/admin/manage/identity/emailmgt/delete <br>
+        <b>Scopes required:</b><br>
+        * internal_email_mgt_delete
+      parameters:
+        - $ref: '#/components/parameters/templateTypeIdPathParam'
+      responses:
+        '204':
+          $ref: '#/components/responses/Deleted'
+        '400':
+          $ref: '#/components/responses/InvalidInput'
+        '401':
+          $ref: '#/components/responses/Unauthorized'
+        '403':
+          $ref: '#/components/responses/Forbidden'
+        '404':
+          $ref: '#/components/responses/NotFound'
+        '500':
+          $ref: '#/components/responses/ServerError'
+  '/email/template-types/{template-type-id}/org-templates':
```

Comment:
Check if we can move this to a bulk endpoint or remove this. Same with the app-templates path.

Reply:
Removed from https://github.com/wso2/identity-api-server/pull/595/commits/5ccba2c271c0d5ccfed72be993661c0c364df066